README.md

```diff
@@ -471,6 +471,49 @@ Allow HTML inside the help text if set to `true`. Default is `false`.
 
 Display help text inline if set to `true`. Default is `false`.
 
+Action API Endpoints
+====================
+
+The extension adds helper methods which expose any configured schemas via:
```

Comment:
these are action endpoints, *not* helper functions.

There are also helper functions defined in https://github.com/ckan/ckanext-scheming/blob/master/ckanext/scheming/helpers.py

Reply:
Ah yeah I'm wrongly using those terms interchangeably here. Amend incoming 😅